ui/src/app/add/components/endpoint-item/endpoint-item.component.ts

```diff
@@ -47,12 +48,19 @@ export class EndpointItemComponent implements OnInit {
     iconReady = false;
     iconError = false;
 
+    cssMapper: Record<string, string> = {
+        ADAPTER: 'adapter',
+        DATA_STREAM: 'stream',
+        DATA_PROCESSOR: 'sepa',
```

Comment:
Is it possible to replace sepa with processor? If the changes would be too big we should create a PR for this task

Reply:
That's a larger undertaking we can address in a separate PR